src/sidebar/store/test/use-store-test.js

```diff
@@ -82,7 +82,10 @@ describe('sidebar/store/use-store', () => {
       assert.deepEqual(proxy.getThing('bar'), { id: 'bar' });
 
       // Test proxied action dispatch.
-      proxy.addThing('baz');
+      act(() => {
+        proxy.addThing('baz');
+      });
```

Comment:
This change is unrelated to the rest of the PR, it aims to fix a flakey test I encountered a couple of times while working on this branch. See https://github.com/hypothesis/client/pull/4245/commits/cba6232848412384525c5506f6265ead72bd8772.